server/portal/apps/webhooks/views.py

```diff
@@ -156,83 +151,44 @@ def post(self, request, *args, **kwargs):
 
         """
         event_type = request.POST.get('event_type', None)
-        if event_type == 'WEB':
-            # This is for jobs that just point to a URL that gets created
-            # like the Potree Viewer Application or DCV-based apps
-            job_owner = request.POST.get('owner', '')
-            address = request.POST.get('address', '')
-            job_uuid = request.POST.get('job_uuid', '')
-            event_data = {
-                Notification.EVENT_TYPE: 'interactive_session_ready',
-                Notification.STATUS: Notification.INFO,
-                Notification.OPERATION: 'web_link',
-                Notification.USER: job_owner,
-                Notification.MESSAGE: 'Ready to view.',
-                Notification.ACTION_LINK: address
-            }
-        elif event_type == 'VNC':
-
-            job_owner = request.POST.get('owner', '')
-            host = request.POST.get('host', '')
-            port = request.POST.get('port', '')
-            password = request.POST.get('password', '')
-            job_uuid = password
-
-            target_uri = \
-                'https://tap.tacc.utexas.edu/noVNC/?'\
-                'host={host}&port={port}&autoconnect=true&encrypt=true&resize=scale&password={pw}' \
-                .format(host=host, port=port, pw=password)
-
-            event_data = {
-                Notification.EVENT_TYPE: 'interactive_session_ready',
-                Notification.STATUS: Notification.INFO,
-                Notification.OPERATION: 'vnc_session_start',
-                Notification.USER: job_owner,
-                Notification.MESSAGE: 'Your VNC session is ready.',
-                Notification.ACTION_LINK: target_uri
-            }
-
-        else:
-            logger.info("Unexpected event type")
-            return HttpResponse("ERROR", status=400)
+        job_uuid = request.POST.get('job_uuid', None)
+        job_owner = request.POST.get('owner', None)
```

Comment:
The job post coming in here seems incorrect as the owner/job_uuid is empty 🤔 .  I'm getting the following which ends up causing an error during the job validation part:

```{'event_type': 'interactive_session_ready', 'address': 'https://frontera.tacc.utexas.edu:60723', 'owner': '', 'job_uuid': ''}```

Reply:
running now!  thanks for looking into it with me.

Reply:
so just follow on issue for `license`

Reply:
Just removed from job submission for now, and added a note :)
https://github.com/TACC/Core-Portal/pull/764/commits/03a7f7aa4058e3a89b4d5f131387ef40d1fa0530

Reply:
[Jira task](https://jira.tacc.utexas.edu/browse/TV3-114)